src/wp-includes/option.php

```diff
@@ -324,36 +324,20 @@ function wp_load_alloptions( $force_cache = false ) {
  * Loads and caches certain often requested site options if is_multisite() and a persistent cache is not being used.
  *
  * @since 3.0.0
- *
- * @global wpdb $wpdb WordPress database abstraction object.
+ * @since 6.1.0 Uses update_meta_cache
```

Comment:
As per the [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs).

> PHPDoc supports multiple `@since` versions in DocBlocks for this explicit reason. When adding changelog entries to the `@since` block, a version should be cited, and a description should be added in sentence case and form and **end with a period:**

`* @since 6.1.0 Now uses update_meta_cache().`

Reply:
@JJJ Thanks for address other feedback. It seems this change does not added in PR. If i add suggestion for this then it include any third person profile for `@since` tag